app/models.py

```diff
@@ -1467,6 +1467,7 @@ def serialize(self):
         data.update({
             'id': self.id,
             'briefId': self.brief_id,
+            'briefTitle': self.brief.data.get('title', ''),
```

Comment:
Feels like this should be a shared Serializer class, that knows about how we want to serialize our sql alchemy things, and could be told in a consistent way whether we want to include related objects, and if so, how far down to recurse.

There is sane-looking prior art for this e.g. the 'recursive, possibly-circular, selective implementation' at https://stackoverflow.com/questions/5022066/how-to-serialize-sqlalchemy-result-to-json

Someone else on there is mentioning Marshmallow also: https://marshmallow-sqlalchemy.readthedocs.io/en/latest/

If we're in a hurry then of course fine to follow the existing pattern, but it just doesn't seem ideal to me.